README.md

```diff
@@ -60,7 +60,7 @@ import Component from '@glimmer/component';
 import { compileHBS } from 'ember-repl';
 
 export class Renderer extends Component {
-  myComponent = compileHBS(this.args.input);
```

Comment:
Hmm then again maybe preserving the complete output of `compileHBS` and just using the part of the object that you need at respective places would be better? 🤔

Reply:
It seems demos / examples tend to be minimal, and real code tends to cover all the states. But, I'm not sure what's better 🤷 I can be convinced either way. 

you could even do:
```js
class Renderer extends Component {
  @cached
  get result() {
    return compileHBS(this.args.input;
  }
  
  get component () {
    return this.result.component;
  }
  
  get error() {
    return this.result.error;
  }
  
  get name() {
    return this.result.name;
  }
}
```
that's probably the _most_ verbose way to get all the info. I'd probably omit `component`, `error`, and `name` altogether and reference the full `this.result.{whatever}` in the template

I kinda wish JS had a shorthand for this syntax. maybe we can pair up with @Rich-Harris (Svelte), and propose something similar to [Svelte's shorthand](https://svelte.dev/examples#reactive-declarations) to TC39:

```js
// imaginary future syntax equivalent shorthand of the above
class Renderer extends Component {
  @cached result := compileHBS(this.args.input);
  
  component := this.result.component;
  error := this.result.error;
  name := this.result.name;
}
```